'println' @ [27:5] ==> public fun println(message: Any?): Unit defined in kotlin.io[DeserializedSimpleFunctionDescriptor]

'+' @ [27:13] ==> public final operator fun plus(other: Any?): String defined in kotlin.String[DeserializedSimpleFunctionDescriptor]

'block' @ [27:26] ==> value-parameter block: () -> Unit defined in kotlin.test.todo[ValueParameterDescriptorImpl]

'assertFails' @ [33:21] ==> @SinceKotlin public fun assertFails(message: String?, block: () -> Unit): Throwable defined in kotlin.test in file Assertions.kt[SimpleFunctionDescriptorImpl]

'message' @ [33:33] ==> value-parameter message: String? defined in kotlin.test.assertFailsWith[ValueParameterDescriptorImpl]

'block' @ [33:42] ==> value-parameter block: () -> Unit defined in kotlin.test.assertFailsWith[ValueParameterDescriptorImpl]

'Suppress' @ [34:5] ==> public constructor Suppress(vararg names: String) defined in kotlin.Suppress[DeserializedClassConstructorDescriptor]

'assertTrue' @ [35:5] ==> public fun assertTrue(actual: Boolean, message: String? = ...): Unit defined in kotlin.test in file Assertions.kt[SimpleFunctionDescriptorImpl]

'exceptionClass' @ [35:16] ==> value-parameter exceptionClass: KClass<T> defined in kotlin.test.assertFailsWith[ValueParameterDescriptorImpl]

'isInstance' @ [35:31] ==> @SinceKotlin public abstract fun isInstance(value: Any?): Boolean defined in kotlin.reflect.KClass[DeserializedSimpleFunctionDescriptor]

'exception' @ [35:42] ==> val exception: Throwable defined in kotlin.test.assertFailsWith[LocalVariableDescriptor]

'messagePrefix' @ [35:54] ==> internal fun messagePrefix(message: String?): String defined in kotlin.test in file Utils.kt[SimpleFunctionDescriptorImpl]

'message' @ [35:68] ==> value-parameter message: String? defined in kotlin.test.assertFailsWith[ValueParameterDescriptorImpl]

'exceptionClass' @ [35:106] ==> value-parameter exceptionClass: KClass<T> defined in kotlin.test.assertFailsWith[ValueParameterDescriptorImpl]

'exception' @ [35:144] ==> val exception: Throwable defined in kotlin.test.assertFailsWith[LocalVariableDescriptor]

'Suppress' @ [37:5] ==> public constructor Suppress(vararg names: String) defined in kotlin.Suppress[DeserializedClassConstructorDescriptor]

'exception' @ [38:12] ==> val exception: Throwable defined in kotlin.test.assertFailsWith[LocalVariableDescriptor]

'DefaultJsAsserter' @ [45:48] ==> internal object DefaultJsAsserter : Asserter defined in kotlin.test in file DefaultJsAsserter.kt[FakeCallableDescriptorForObject]

